README.org

```diff
@@ -23,119 +25,187 @@ is known.  You can also add an entry for *yourself* if you don't accept
+
+*** Donate
+
+-  https://gratipay.com/~bbatsov/
+-  https://gratipay.com/cider
+
+*** Projects
+
+-  [[https://github.com/clojure-emacs/cider][cider]]
+-  [[https://github.com/bbatsov/projectile][projectile]]
+
+** John Miller
+
+-  [[https://github.com/millejoh][GitHub]]
+-  [[https://millejoh.github.io/ (to which I never write)][Blog]]
+
+*** Donate
+
+-  [[https://gratipay.com/Emacs-IPython-Notebook/][Gratipay]]
+
+*** Projects
+
+-  [[https://github.com/millejoh/emacs-ipython-notebook][emacs-ipython-notebook]]
+
+** Jonas Bernoulli
+
+-  [[https://github.com/tarsius][GitHub]]
+-  [[https://twitter.com/magit_emacs][Twitter]]
+-  [[https://emacsair.me][Blog]]
+
+*** Donate
+
+-  https://magit.vc/donate
+-  bitcoin: 1KvLZpbo1AZCPsMJidUyLxS44uXGBxxHgK
+-  [[https://www.bountysource.com/teams/magit][bountysource]]
+-  ethereum: 0x90c98E49862d2Ac495BCA94342426D4e90286Ef5
+-  gratipay
+   -  https://gratipay.com/~tarsius
+   -  https://gratipay.com/magit
+-  [[https://liberapay.com/tarsius][liberapay]]
+-  [[https://opencollective.com/magit][opencollective]]
+-  [[https://www.patreon.com/tarsius][patreon]]
+-  [[https://www.paypal.me/JonasBernoulli][paypal]]
+
+*** Projects
+
+-  [[https://github.com/emacscollective][auto-compile, borg, epkg, no-littering]]
+-  [[https://emacsmirror.net][emacsmirror]]
+-  [[https://magit.vc][magit]]
+
+** Matus Goljer
+
+-  [[https://github.com/Fuco1][GitHub]]
+-  [[https://fuco1.github.io/][Blog]]
+
+*** Donate
+
+-  [[https://www.patreon.com/user?u=3282358][Patreon]]
+-  [[https://www.paypal.me/MatusGoljer][PayPal]]
+
+*** Projects
+
+-  [[https://github.com/Fuco1/smartparens][smartparens]]
+-  [[https://github.com/Fuco1/dired-hacks][dired-hacks]]
+-  [[https://github.com/magnars/dash.el][dash]]
+-  [[https://github.com/Fuco1/litable][litable]]
+
+** Nicolas Petton
+
+-  [[https://github.com/NicolasPetton][GitHub]]
+-  [[https://twitter.com/NicolasPetton][Twitter]]
+-  [[https://nicolas.petton.fr][Blog]]
+
+*** Donate
+
+-  [[https://liberapay.com/NicolasPetton][liberapay]]
+
+** Oleh Krehel
+
+-  [[https://github.com/abo-abo][GitHub]]
+-  [[https://twitter.com/_abo_abo][Twitter]]
+-  [[http://oremacs.com][Blog]]
+   -  [[https://oremacs.com/2017/10/18/patreon][Please consider supporting me on Patreon]]
+   -  [[https://oremacs.com/2017/12/10/patreon][Comparison of transaction fees on Patreon and similar services]]
+
+*** Donate
+
+-  [[https://flattr.com/@abo-abo][flattr]]
+-  [[https://liberapay.com/abo-abo][liberapay]]
+-  [[https://www.patreon.com/abo_abo][patreon]]
+-  [[https://www.paypal.me/aboabo][paypal]]
+
+** Steve Purcell
+
+-  [[https://github.com/purcell][GitHub]]
+-  [[https://twitter.com/sanityinc][Twitter]]
+-  [[http://www.sanityinc.com][Blog]]
+
+*** Donate
+
+-  [[https://www.patreon.com/sanityinc][patreon]]
+
+*** Projects
+
+-  [[https://melpa.org][melpa]]
+-  [[https://github.com/purcell/package-lint][package-lint]]
+
+** Ted Zlatanov <tzz@lifelogs.com>
```

Comment:
Use a separate list item for the email, like for the github page.

Reply:
Good idea.